chdconvert.py

```diff
@@ -6,12 +6,15 @@
 
 input_dir = input("Input Dir:\n") if len(sys.argv) < 2 else sys.argv[1]
 tmp_dir = input_dir + "_tmp"
-chdman = os.path.join(os.path.dirname(__file__), 'chdman.exe')
+if os.name == 'nt':
+    chdman = os.path.join(os.path.dirname(__file__), 'chdman.exe')
+else:
+    chdman = 'chdman'
 operator = sys.argv[2] if (len(sys.argv) == 3 and sys.argv[2] in ["-d", "--delete", "-r", "--replace"]) else None
 error_log_file = "skipped_archives.txt"
 
 # Updated to handle both gz and 7z files
-incoming_files = [f for f in os.listdir(input_dir) if f.split(".")[-1] in ["gz", "7z"]]  
+incoming_files = [f for f in os.listdir(input_dir) if f.split(".")[-1] in ["gz", "7z", "zip"]]  # Support additional archives, eg. zip

```

Comment:
Adding zip was a safe bet. I wonder what other formats we could/should add. 

`Pyunpack` is the module used for these, and it supports just about every file format - though it looks like it also requires existing system dependencies for each format besides zip. [pyunpack](https://github.com/ponty/pyunpack)

In any case, im fine with this if you are. Thanks!